File: app/src/main/java/kr/djspi/pipe01/dto/SpiLocation.java
package kr.djspi.pipe01.dto;

import java.io.Serializable;

import lombok.Data;

@Data
public class SpiLocation implements DataItem, Serializable {

    public int id = -1;
    public int spi_id;
    /**
     * 위도(latitude), 경도(longitude) 좌표
     */
    public double latitude;
    public double longitude;
    /**
     * count 값은 사용자가 직접 입력하지 않는 값
     */
    public int count;

    public void setCount(int i) {
        this.count = i;
        this.count++;
    }
}

File: app/src/main/java/kr/djspi/pipe01/dto/SpiLocation.kt
package kr.djspi.pipe01.dto

import java.io.Serializable

data class SpiLocation(var id: Int = -1) : DataItem, Serializable {

    var spi_id: Int? = null
    var latitude: Double? = null
    var longitude: Double? = null
    var coordinate_x: Double? = null
    var coordinate_y: Double? = null
    var origin: String? = null
    var count: Int = 0
        set(value) {
            field = value
            field++
        }
}
